internal/app/configurable.go

```diff
@@ -21,85 +21,96 @@ import (
-	CompressZLIB        = "zlib"
-	EncryptAES256       = "aes256"
-	Mode                = "mode"
-	BatchByCount        = "bycount"
-	BatchByTime         = "bytime"
-	BatchByTimeAndCount = "bytimecount"
-	IsEventData         = "iseventdata"
-	MergeOnSend         = "mergeonsend"
-	HttpRequestHeaders  = "httprequestheaders"
-	WillEnabled         = "willenabled"
-	WillTopic           = "willtopic"
-	WillQos             = "willqos"
-	WillPayload         = "willpayload"
-	WillRetained        = "willretained"
+	ProfileNames            = "profilenames"
+	DeviceNames             = "devicenames"
+	SourceNames             = "sourcenames"
+	ResourceNames           = "resourcenames"
+	FilterOut               = "filterout"
+	EncryptionKey           = "key"
+	InitVector              = "initvector"
+	Url                     = "url"
+	ExportMethod            = "method"
+	ExportMethodPost        = "post"
+	ExportMethodPut         = "put"
+	MimeType                = "mimetype"
+	PersistOnError          = "persistonerror"
+	ContinueOnSendError     = "continueonsenderror"
+	ReturnInputData         = "returninputdata"
+	SkipVerify              = "skipverify"
+	Qos                     = "qos"
+	Retain                  = "retain"
+	AutoReconnect           = "autoreconnect"
+	ConnectTimeout          = "connecttimeout"
+	ProfileName             = "profilename"
+	DeviceName              = "devicename"
+	ResourceName            = "resourcename"
+	ValueType               = "valuetype"
+	MediaType               = "mediatype"
+	Rule                    = "rule"
+	BatchThreshold          = "batchthreshold"
+	TimeInterval            = "timeinterval"
+	HeaderName              = "headername"
+	SecretName              = "secretname"
+	SecretValueKey          = "secretvaluekey"
+	BrokerAddress           = "brokeraddress"
+	ClientID                = "clientid"
+	KeepAlive               = "keepalive"
+	Topic                   = "topic"
+	TransformType           = "type"
+	TransformXml            = "xml"
+	TransformJson           = "json"
+	AuthMode                = "authmode"
+	Tags                    = "tags"
+	ResponseContentType     = "responsecontenttype"
+	Algorithm               = "algorithm"
+	CompressGZIP            = "gzip"
+	CompressZLIB            = "zlib"
+	EncryptAES256           = "aes256"
+	Mode                    = "mode"
+	BatchByCount            = "bycount"
+	BatchByTime             = "bytime"
+	BatchByTimeAndCount     = "bytimecount"
+	IsEventData             = "iseventdata"
+	MergeOnSend             = "mergeonsend"
+	HttpRequestHeaders      = "httprequestheaders"
+	WillEnabled             = "willenabled"
+	WillTopic               = "willtopic"
+	WillQos                 = "willqos"
+	WillPayload             = "willpayload"
+	WillRetained            = "willretained"
+	MaxReconnectInterval    = "maxreconnectinterval"
+	PreConnect              = "preconnect"
+	PreConnectRetryCount    = "preconnectretrycount"
+	PreConnectRetryInterval = "preconnectretryinterval"
+
```

Comment:
Is the new line here intentional?

Reply:
yes, following constants are different type.